kotest-common/src/commonMain/kotlin/io/kotest/common/annotations.kt

```diff
@@ -16,7 +16,12 @@ annotation class DelicateKotest
 /**
  * An internal Kotest feature that is public for operational reasons but should not be used by end users.
  */
-@Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION, AnnotationTarget.TYPEALIAS)
+@Target(
+   AnnotationTarget.CLASS,
+   AnnotationTarget.FUNCTION,
+   AnnotationTarget.TYPEALIAS,
+   AnnotationTarget.PROPERTY,
```

Comment:
You didn't actually add the annotation on a property in the end? Should we revert the change here?

Reply:
`AnnotationTarget.PROPERTY` is required for adding `@KotestInternal` to `val start` in `logger.kt`

Reply:
Oh, right :) carry on